src/preview.ts

```diff
@@ -10,21 +10,11 @@ export function debug(element: Element = document.body): void {
     });
   }
 
-  // If user use styled-components
-  if (getStyle) {
```

Comment:
Can we safely remove `./styled-components` now?

Reply:
For sure. I will remove that.
 We don't need to access to `styled-component`'s internal to make jest preview works anymore. But at least I learned a lot of how `styled-components` works